js/whitebit.js

```diff
@@ -1194,68 +1195,83 @@ module.exports = class whitebit extends Exchange {
         //
-        //     {
-        //         "market": "BTC_USDT"
-        //         "amount": "0.0009",
-        //         "price": "40000",
-        //         "type": "limit",
-        //         "id": 4986126152,
-        //         "clientOrderId": "customId11",
-        //         "side": "sell",
-        //         "ctime": 1597486960.311311,       // timestamp of order creation
-        //         "takerFee": "0.001",
-        //         "ftime": 1597486960.311332,       // executed order timestamp
-        //         "makerFee": "0.001",
-        //         "dealFee": "0.041258268",         // paid fee if order is finished
-        //         "dealStock": "0.0009",            // amount in stock currency that finished
-        //         "dealMoney": "41.258268"          // amount in money currency that finished
-        //     }
+        //      {
+        //          "id":105531094719,
+        //          "clientOrderId":"",
+        //          "ctime":1659045334.550127,
+        //          "ftime":1659045334.550127,
+        //          "side":"buy",
+        //          "amount":"5.9940059",           // cost in terms of quote for regular market orders, amount in terms or base for all other order types
+        //          "price":"0",
+        //          "type":"market",
+        //          "takerFee":"0.001",
+        //          "makerFee":"0",
+        //          "dealFee":"0.0059375815",
+        //          "dealStock":"85",               // base filled amount
+        //          "dealMoney":"5.9375815",        // executed amount in quote
+        //      }
         //
         const marketId = this.safeString (order, 'market');
         market = this.safeMarket (marketId, market, '_');
         const symbol = market['symbol'];
         const side = this.safeString (order, 'side');
         const filled = this.safeString (order, 'dealStock');
-        const remaining = this.safeString (order, 'left');
+        let remaining = this.safeString (order, 'left');
         const clientOrderId = this.safeString (order, 'clientOrderId');
         let price = this.safeString (order, 'price');
-        const stopPrice = this.safeString (order, 'activation_price');
+        const stopPrice = this.safeNumber (order, 'activation_price');
         const orderId = this.safeString2 (order, 'orderId', 'id');
-        const type = this.safeString (order, 'type');
+        const rawType = this.safeString (order, 'type');
         let amount = this.safeString (order, 'amount');
         let cost = undefined;
         if (price === '0') {
             // api error to be solved
             price = undefined;
         }
-        if (side === 'buy' && type.indexOf ('market') >= 0) {
+        let timeInForce = undefined;
+        if (rawType === 'stock market') {
+            timeInForce = 'FOK';
+        }
+        const type = this.parseOrderType (rawType);
+        if (side === 'buy' && type === 'market') {
```

Comment:
@DoctorSlimm, actually we can't do that here because "stock market" will also translate to "market" and for this type this does not apply, as you can see here
![image](https://user-images.githubusercontent.com/43336371/182580273-5b14ba81-b182-4ad7-9223-71d85f47ac30.png)

I think it's easier to do this: `if (side === 'buy' && type === 'market' || type==='stop_market') {`

Reply:
My bad, got so confused with all the merge conflicts. Will add it back in today.

Reply:
@DoctorSlimm no problem 🙂